components/team_sidebar/team_sidebar.tsx

```diff
@@ -305,14 +305,17 @@ export default class TeamSidebar extends React.PureComponent<Props, State> {
             );
         }
 
-        plugins.push(
-            <div
-                key='team-sidebar-bottom-plugin'
-                className='team-sidebar-bottom-plugin is-empty'
-            >
-                <Pluggable pluggableName='BottomTeamSidebar'/>
-            </div>,
-        );
+        // Disable team sidebar pluggables in products until proper support can be provided.
+        if (!currentProduct) {
```

Comment:
Ah, is this because channels doesn't yet register itself as a product? Might be interesting to declare a variable `isNonChannelsProduct` to help clarify, but 0/5.

Reply:
I agree with @lieut-data even something like a line before the if with `const isNonChannelsProduct = !currentProduct;` and use that variable in  the if would explain way better the intention. Anyway, not a blocker, for sure.

Reply:
Agree. Done.